flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -41,57 +42,12 @@
 #include "llvm/Frontend/OpenMP/OMPConstants.h"
 
 using namespace Fortran::lower::omp;
+using namespace Fortran::common::openmp;
```

Comment:
Nit: For consistency, it looks like `Fortran::common::omp` would be a better name for the namespace in the end.